software/desktop-environments/en.md

```diff
@@ -30,4 +30,22 @@ Solus offers the MATE desktop environment / experience, tailored to advanced use
 
 ``` bash
 sudo eopkg install -c desktop.mate
-```
\ No newline at end of file
+```
+
+## Plasma
+
+Solus will offer the KDE Plasma desktop environment / experience in the next comming release. Plasma is a powerful and highly customizable desktop enviroment built with Qt and the KDE framework. It is reccomended to use the ISO but until then, you can try it right now by running the following command:
+
+``` bash
+sudo eopkg install -c desktop.kde.core
+```
+
+## i3
+
+Solus offers i3 as a tiling window manager for advanced users that seek window manager instead of a full desktop environment. i3 uses keyboard shortcuts to split windows either horizontally or vertically. Users wanting to try i3 can install it by running the following command:
```

Comment:
So aside from "that seek window manager" not being correct English (in addition to being redundant), I disagree with pitching i3 strictly as something for "advanced users" (what would we classify as advanced, anyways?). I'd simply state:

    Solus offers i3 as a tiling window manager. Users wanting to try i3 can install it by running the following command:

    ``` bash
    sudo eopkg install i3
    ```

    If you want to use i3 with support for GNOME Settings Daemon and other GNOME functionality, follow the instructions for installation of GNOME Shell, then choose GNOME+i3 at the login screen to use it.

    For more information about tiling window managers, click [here](https://en.wikipedia.org/wiki/Tiling_window_manager).

Reply:
Yeah I see, fixed now!